<launch>
    <!-- Openni_Tracker Node-->

    <node pkg="openni_tracker" name="openni_tracker" type="openni_tracker" output="screen">
        <param name="fixed_frame" value="openni_depth_frame" />
    </node>

    <!-- Turtlesim Node-->
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
    <node pkg="move_turtle" type="move_sim.py" 
          name="listener" />

    <node name="turtle1_tf_broadcaster" pkg="move_turtle" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle1" />
    </node>
    
    <!--<node name="turtle_tracker" pkg="move_turtle" type="turtle_tracker.py"/>-->
    
    <!--<node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>

    <node pkg="move_turtle" type="turtle_tf_listener.py" 
          name="listener" />

    <node name="turtle1_tf_broadcaster" pkg="move_turtle" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle1" />
    </node>

    <node name="turtle2_tf_broadcaster" pkg="move_turtle" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle2" /> 
    </node>-->
</launch>